packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -9,6 +9,13 @@ it('should render a default radio field', () => {
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render a default radio field without crashing if it has a null child', () => {
```

Comment:
either change this to 
```suggestion
it('should render a default radio field without crashing if it has an undefined child', () => {
```

Reply:
Updated: efc1d732b